feature-libs/cart/import-export/components/export-entries/export-entries.component.ts

```diff
@@ -19,29 +19,9 @@ export class ExportEntriesComponent {
   entries$ = this.exportEntriesService.getEntries();
 
   exportToCsv() {
-    this.entries$
-      .pipe(take(1))
-      .subscribe((entries) => this.parseEntries(entries));
-  }
-
-  parseEntries(entries: OrderEntry[]) {
-    let parsedData = [];
-    parsedData.push({
-      sku: 'Sku',
-      quantity: 'Quantity',
-      name: 'Name',
-      price: 'Price',
-    });
-    entries.forEach((element: OrderEntry) => {
-      parsedData.push({
-        sku: element?.product?.code,
-        quantity: element?.quantity,
-        name: element?.product?.name,
-        price: element?.totalPrice?.formattedValue,
-      });
-    });
-
-    this.downloadCsv(this.exportService.dataToCsv(parsedData));
+    this.downloadCsv(
+      this.exportService.dataToCsv(this.exportEntriesService.exportEntries())
+    );
   }
 
   downloadCsv(csvData: any, filename = 'data') {
```

Comment:
```suggestion
  downloadCsv(csvData: string, filename = 'data') {
```
`filename = 'data'` - what do you think about use cart name here?

Reply:
ok, regarding to filename it's a scope of another ticket